sys/net/gnrc/sock/include/sock_types.h

```diff
@@ -27,8 +27,6 @@
 
 #include "mbox.h"
 #include "net/af.h"
-#include "net/gnrc.h"
-#include "net/gnrc/tcp.h"
```

Comment:
Now this breaks the [sock TCP example](https://doc.riot-os.org/group__net__sock__tcp.html)

Reply:
sorry, good that you cought it and fixed it with #16723, which will also be a good check to avoid this happening again

Reply:
tbh I thought #16739 would be less controversial way to get a (compile) test for a generic sock TCP but turns out that was not as easy as I thought